test/functionalTest/cases/0985_DELETE_v2_entity/delete_ok.test

```diff
@@ -0,0 +1,127 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+DELETE Entity V2
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 51
+
+--SHELL--
+
+#
+# 01. POST /v2/entities
+# 02. GET /v2/entities/E1
+# 03. DELETE /v2/entities/E1
+# 04. GET /v2/entities/E1
+#
```

Comment:
I would like to add one more DELETE /v2/entities/E1, just to see how it handles 'entity not found'